<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_x">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>31</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="m_widget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>1191</width>
      <height>551</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Precision Sans W Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Precision Sans W Medium</family>
     </font>
    </property>
    <property name="text">
     <string>Отрисовка</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_width">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Толщина:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fill">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Заливка:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_fill">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Да</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_style">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Стиль:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Precision Sans W Medium</family>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Пиксель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Линия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прямоугольник</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Окружность</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эллипс</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дуга</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ломанная линия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Многоугольник</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_styleb">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Стиль заливки:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_style">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NoPen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SolidLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DashLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DotLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DashDotLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DashDotDotLine</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_styleb">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SolidPattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dense1Pattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dense2Pattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dense3Pattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dense4Pattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dense5Pattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dense6Pattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dense7Pattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HorPattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VerPattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CrossPatern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BDiagPattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FDiagPattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DiagCrossPattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LinearGradientPattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RadialGradientPattern</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ConicalGradientPattern</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_startangle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_spanangle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_x2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Гор. радиус</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Гор. радиус</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sa">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Нач. угол</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_spa">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Конечный угол</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>m_widget</class>
   <extends>QWidget</extends>
   <header>m_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
